action.yaml

```diff
@@ -114,6 +114,8 @@ runs:
     - ${{ inputs.include }}
     - "--subsystem"
     - ${{ inputs.subsystem }}
+    - "--convert"
+    - ${{ inputs.convert }}
```

Comment:
The `convert` input is not required, see line 48. This raises the question: what would the command line look like if the user indeed does not provide a value for `convert`. Would GitHub add just `--convert` to the command line? If so, how would the Python script deal with that?